tables/scripts/ptrepack.py

```diff
@@ -422,8 +422,15 @@ def main():
         )
 
     # Catch the files passed as the last arguments
-    src = args.src.split(':')
-    dst = args.dst.split(':')
+    if 'win' in sys.platform():
+        # the file separator for windows is deemed ','
+        src = args.src.split(',')
+        dst = args.dst.split(',')
+    else:
+        # the file separator for osx, linux for  is deemed ':'
+        src = args.src.split(':')
```

Comment:
What about: `src = args.src.rsplit(':', 1)`.

That should work on windows and posix.